accounts/viewsets.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework.viewsets import ModelViewSet
+from .models import User
+from .serializers import UserSerializer
+
+
+class UserViewSet(ModelViewSet):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
```

Comment:
can we also add `IsAuthenticated` to the `permission_classes`?

Reply:
added :)